netdata/netdata
# Fix SSL renegotiation errors in old versions of openssl.

##### Summary
Fix SSL renegotiation errors in old versions of openssl.

##### Component Name
aclk

##### Test Plan

* Setup nodefarm - connects variety of nodes to cloud staging.
* Use `vagrant suspend` to pause the machine long enough to become unreachable
* Use `vagrant resume` to un-pause the VM and force reconnection.

Run the agent through valgrind and check the shutdown sequence.

##### Additional Information

Not ready for review yet, need to rework the shutdown sequence and do some more testing.
